docs/development.md

```diff
@@ -0,0 +1,23 @@
+```{include} ../CONTRIBUTING.md
+```
+
+## Contributing documentation
+
+### Building the documentation locally (optional)
+
+If you are writing documentation for MaxText, you may want to preview the documentation site locally. First, make sure you install the necessary dependencies. You can do this by navigating to your local clone of the MaxText repo and running:
+```bash
+pip install -r requirements_docs.txt
+```
+
+Once the dependencies are installed, you can navigate to the `docs/` folder and run:
+
+```bash
+sphinx-build -b html . _build/html
+```
+
+This will generate the documentation in the `docs/_build/html` directory.
```

Comment:
```suggestion
This will generate the documentation in the `docs/_build/html` directory.

If you are adding a new document, make sure it is included in the `toctree` directive corresponding to the section where the new document should live. For example, if adding a new tutorial, make sure it is listed in [the `docs/tutorials.md`](https://github.com/AI-Hypercomputer/maxtext/blob/7070e8eecbea8951c8e5281219ce797c8df1441f/docs/tutorials.md?plain=1#L38).
```